File: common-basic/src/main/java/com/dosmono/sanya/basic/router/RouterParty.java

package com.dosmono.sanya.basic.router;
public interface RouterParty {
    /**
     * 组名
     */
    String APP = "/app";//宿主 App 组件
    String TIANYA = "/tianya";//天涯组件
    String JIYANG = "/jiyang";//吉阳组件

    /**
     * 服务组件, 用于给每个组件暴露特有的服务
     */
    String SERVICE = "/service";


    /**
     * 宿主 App 分组
     */
    String APP_MAINACTIVITY = APP + "/MainActivity";


    /**
     * 天涯分组
     */
    String TIANYA_SERVICE_TIANYAINFOSERVICE = TIANYA + SERVICE + "/TIANYAInfoService";

    String TIANYA_HOMEACTIVITY = TIANYA + "/HomeActivity";

    /**
     * 吉阳分组
     */
    String JIYANG_SERVICE_JIYANGINFOSERVICE = JIYANG + SERVICE + "/JIYANGInfoService";

    String JIYANG_HOMEACTIVITY = JIYANG + "/HomeActivity";

}

File: lib-component/src/main/java/com/dosmono/sanya/component/RouterParty.kt
package com.dosmono.sanya.component

object RouterParty {


    const val SERVICE = "/service"

    //宿主 App 组件
    object App {
        private const val APP_GROUP = "/app"

    }

    //main组件
    object Main {
         const val GROUP = "/main"

        const val MAIN_ACTIVITY = "$GROUP/mainactivity"
    }

    // sub组件
    object Sub {
         const val GROUP = "/sub"
        const val SUB_ACTIVITY = "$GROUP/subactivity"
    }


    // talk组件
    object Talk {
        const val GROUP = "/talk"
        const val TALK_ACTIVITY = "$GROUP/talkactivity"
    }


}